app/views/devise/registrations/edit.html.erb

```diff
@@ -30,6 +30,57 @@
     <%= f.label :current_password %> <i>(we need your current password to confirm your changes)</i><br />
     <%= f.password_field :current_password, autocomplete: "current-password" %>
   </div>
+  <br />
+  <!--Personal Info from new user-->
+  <div class="col-8">
+  <div class="card text-white bg-info mb-3" style="max-width: 50rem;">
+    <div class="card-header">User Information</div>
+    <div class="card-body">  
+      <h5 class="card-title">User</h5>
+        <div>
+             <%= f.label :picture %>
+             <%= f.text_field :picture, autofocus: true %>
+        </div>
+        <div>
+             <%= f.label :first_name %>
+             <%= f.text_field :first_name, autofocus: true %>
+        
+             <%= f.label :last_Name %>
+             <%= f.text_field :last_name, autofocus: true %>
+        </div>
+        <div>
+             <%= f.label :Gender %>
+             <%= f.select :gender,[ 'Female', 'Male', 'Other', 'N/A' ], :prompt => 'Select One', autofocus: true %>
+        </div>   
+        <div>
+             <%= f.label :Age %>
+             <%= f.select :age, options_for_select(18..100), :prompt => "Age" ,autofocus: true %>
+        </div>
+        <div>
+             <%= f.label :Address %>
```

Comment:
The parameter name that you pass to `f.label` needs to *exactly* match the parameter name that you pass to `f.text_field`. In this case, that is `:street_address`. If you want the label to show different text, you can do that easily by passing in an additional parameter to `f.label`:

```suggestion
             <%= f.label :street_address, "Address" %>
```

For more information on this, please see [the Rails documentation](https://apidock.com/rails/ActionView/Helpers/FormHelper/label) and [this Stack Overflow post](https://stackoverflow.com/questions/13003626/custom-text-for-rails-form-for-label).